app/models/user.rb

```diff
@@ -13,7 +13,7 @@ class User < ApplicationRecord
 
   before_create :set_default_role
 
-  validates :name, presence: true
+  # validates :name, presence: true
```

Comment:
This is good, actually. This is a good way to keep this branch scoped to what your changes should be.

However, we obviously will wait to deploy this until the blocking, dependent branches have shipped, so you will end up not needing this!

Reply:
Hi @raquii Before I do this, should I do a git pull?